prow/github/fakegithub/fakegithub.go

```diff
@@ -195,3 +195,7 @@ func (f *FakeClient) GetFile(org, repo, file, commit string) ([]byte, error) {
 
 	return nil, fmt.Errorf("could not find file %s with ref %s", file, commit)
 }
+
+func LabelID(org, repo string, number int, label string) string {
```

Comment:
pulled this out of the functions and exported for use in external libs...right call?